indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -96,8 +96,23 @@
                                 </td>
                                 {% for item in static_columns %}
                                     {% if item.id == 'state' %}
-                                        <td class="i-table" data-searchable="{{ abstract.state.title | lower }}">
-                                            {{- abstract.state.title -}}
+                                        <td class="i-table" data-searchable="{{ abstract.public_state.title | lower }}">
+                                            <div class="i-tag outline semantic-text {{ abstract.public_state.css_class }} state-badge">
+                                                {% if abstract.public_state.name == 'under_review' and abstract.can_judge(session.user) %}
+                                                    {% trans count=abstract.reviews|length -%}
+                                                        {{- count }} review on
```

Comment:
This is not very good i8n since "x review on" and "x tracks" are translated separately. Unfortunately there is no way of doing it better with this structure. However, why not use parentheses instead of "on"?

"X review(s) (Y track(s))"

Like this you can translate/pluralize "X review(s)" and "Y track(s)" separately but it should be ok in any language